README.md

```diff
@@ -5,6 +5,7 @@
 [![PyPi](https://img.shields.io/pypi/v/dag-factory.svg)](https://pypi.org/project/dag-factory/)
 [![Code Style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
 [![Downloads](https://pepy.tech/badge/dag-factory)](https://pepy.tech/project/dag-factory)
+<img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=2bb92a5b-beb3-48cc-a722-79dda1089eda" />
 
 Welcome to *dag-factory*! *dag-factory* is a library for [Apache Airflow®](https://airflow.apache.org) to construct DAGs declaratively via configuration files. 
 
```

Comment:
@cmarteepants, it is important that we say in our docs the notice you wrote in the PR description, referencing
> This adds website analytics to the Dag-Factory readme. Scarf privacy policy: https://about.scarf.sh/privacy-policy

Or https://airflow.apache.org/privacy-notice/ or https://privacy.apache.org/policies/privacy-policy-public.html, which the community may more know.

Reply:
Looking at other apache examples such as [Apache Hudi](https://github.com/apache/hudi/pull/10775/files), I don't see any specific disclaimers to website analytics for readme's. Are there any examples that you can point to where we say that tracking pixels are used in the readme?

Reply:
Apache Airflow docs: https://airflow.apache.org/privacy-notice/ .
We can have a dedicated page for this as well - it's just that ATM in DAG Factory all the doc lives in the README.

Reply:
Added a document that redirects to Astronomer's privacy policy

Reply:
Thanks a lot, @cmarteepants !